packages/coinstac-ui/app/render/state/ducks/util.js

```diff
@@ -1,10 +1,12 @@
 import app from 'ampersand-app';
 
-export const notifyAndThrow = (err) => {
```

Comment:
The linter wants `default` in there for cases when it's the only export: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

I could write it differently, but this way retained the function title, which I thought might be good for self-documenting purposes.